demos/stable-diffusion-1.5/index.js

```diff
@@ -325,41 +326,42 @@ async function readResponse(name, response) {
 }
 
 Utils.log("[Load] Loading ONNX Runtime");
-const progressBarInner = document.getElementById("progress-bar-inner");
-const progressBarLabel = document.getElementById("progress-bar-label");
-const progressBarInnerInference = document.querySelector("#progress-bar-inner-inference");
-const progressBarLabelInference = document.querySelector("#progress-bar-label-inference");
-
-const startButton = document.getElementById("generate_next_image");
-const loadButton = document.getElementById("load_models");
-const positiveInput = document.getElementById("positive_prompt");
-const negativeInput = document.getElementById("negative_prompt");
-const positiveTokenInfo = document.getElementById("positive_token_info");
-const negativeTokenInfo = document.getElementById("negative_token_info");
-const error = document.querySelector("#error");
-const userSeed = document.querySelector("#user_seed");
-const changeSeed = document.querySelector("#change_seed");
-const title = document.querySelector("#title");
-const data = document.querySelector("#data");
-const textEncoderLoad = document.querySelector("#textencoderload");
-const textEncoderFetch = document.querySelector("#textencoderfetch");
-const textEncoderCreate = document.querySelector("#textencodercreate");
-const textEncoderRun = document.querySelector("#textencoderrun");
-const unetLoad = document.querySelector("#unetload");
-const unetFetch = document.querySelector("#unetfetch");
-const unetCreate = document.querySelector("#unetcreate");
-const unetRun = document.querySelector("#unetrun");
-const vaeDecoderLoad = document.querySelector("#vaedecoderload");
-const vaeDecoderFetch = document.querySelector("#vaedecoderfetch");
-const vaeDecoderCreate = document.querySelector("#vaedecodercreate");
-const vaeDecoderRun = document.querySelector("#vaedecoderrun");
-const scTr = document.querySelector("#sc");
-const scLoad = document.querySelector("#scload");
-const scFetch = document.querySelector("#scfetch");
-const scCreate = document.querySelector("#sccreate");
-const scRun = document.querySelector("#scrun");
-const totalLoad = document.querySelector("#totalload");
-const totalRun = document.querySelector("#totalrun");
+const progressBarInner = $("#progress-bar-inner");
+const progressBarLabel = $("#progress-bar-label");
+const progressBarInnerInference = $("#progress-bar-inner-inference");
+const progressBarLabelInference = $("#progress-bar-label-inference");
```

Comment:
Previously there was a call to `getElementById` and `querySelector` above, but the perlish-looking `$` now maps both to `document.querySelector`. Is that correct?

Reply:
@fdwr getElementById() only works with ID attributes, while querySelector() can work with any CSS selector. 

`getElementById("<id>") == querySelector("#<id>").`

```
const $ = s => document.querySelector(s);
const $$ = s => [...document.querySelectorAll(s)];
```

Correct, $ maps to getElementById("<id>") and querySelector("#<id>").